sfdc/Module.md

```diff
@@ -103,205 +31,150 @@ sfdc:SalesforceConfiguration sfConfig = {
+
+  ```ballerina
+  string testRecordId = "001xa000003DIlo";
+  sfdc:Error? isDeleted = baseClient->deleteRecord("Account", testRecordId);
+  ```
+Convenient CRUD operations for common SObjects  
+Apart from the common CRUD operations that can be used with any SObject, the Ballerina Salesforce connector provides customized CRUD operations for pre-identified, most commonly used SObjects. They are **Account**, **Lead**, **Contact**, **Opportunity** and **Product**.   
+Following are the sample codes for Account’s CRUD operations and the other above mentioned SObjects follow the same implementation and only the Id should be changed according to the SObject type. 
+
+- Create account  
+`createAccount` remote function accepts an account record in json as an argument and returns Id of the account created at success. 
+
+  ```ballerina
+  json accountRecord = {
+    Name: "John Keells Holdings",
+    BillingCity: "Colombo 3"
+  };
+
+  string|sfdc:Error accountId = baseClient->createAccount(accountRecord);
+  ```
+
+- Get account by ID  
+User needs to pass the Id of the account and the names of the fields needed parameters for the `getAccountById` remote function. Function will return the record in json format at success. 
+
+  ```ballerina
+  string accountId = "001xa000003DIlo";
+
+  json|sfdc:Error account = baseClient->getAccountById(accountId, Name, BillingCity);
+  ```
+
+- Update account  
+`updateAccount` remote function accepts account id and the account record needed to update in json as arguments and returns true at success. 
+
+  ```ballerina
+  string accountId = "001xa000003DIlo";
+  json account = {
+        Name: "WSO2 Inc",
+        BillingCity: "Jaffna",
+        Phone: "+94110000000"
+    };
+  sfdc:Error? isSuccess = baseClient->updateRecord(accountId, account);
+  ```
+
+- Delete account  
+User needs to pass the Id of the account he needs to delete for the `deleteAccount` remote function. Function will return true at success. 
+
+  ```ballerina
+  string accountId = "001xa000003DIlo";
+  sfdc:Error? isDeleted = baseClient->deleteAccount(accountId);
+  ```
+
+- Query operations  
+The `getQueryResult` remote function executes a SOQL query that returns all the results in a single response or if it exceeds the maximum record limit, it returns part of the results and an identifier that can be used to retrieve the remaining results.
+
+  ```ballerina
+  string sampleQuery = "SELECT name FROM Account";
+  SoqlResult|Error res = baseClient->getQueryResult(sampleQuery);
+  ```
+
+  The response from `getQueryResult` is either a `SoqlResult` record with total size, execution status, resulting records, and URL to get the next record set (if query execution was successful) or Error (if the query execution was unsuccessful).
+
+  ```ballerina
```

Comment:
This is extracting the data from the SOQL right?

Reply:
We decided to remove quick reference from module.md.